scripts/docs/templates/top-config.md.tmpl

```diff
@@ -6,7 +6,7 @@
 {{ range . -}}
 | `{{.yamlName}}` |
 {{- ` `}}{{if .required}}**yes**{{else}}no{{end}} |
-{{- ` `}}{{if has . "elementStruct"}}[{{end}}{{or (index $doc_types .type) .type}}{{if .elementKind}} of {{or (index $doc_types .elementKind) .elementKind}}{{end}}{{if has . "elementStruct"}}](#{{.yamlName | toLower}}){{end}} |
+{{- ` `}}{{if has . "elementStruct"}}[{{end}}{{or (index $doc_types .type) .type}}{{if (eq .type "struct") }} (see below){{end}}{{if .elementKind}} of {{or (index $doc_types .elementKind) .elementKind}}s{{if (index $doc_types .elementKind) "object"}} (see below){{end}}{{end}}{{if has . "elementStruct"}}](#{{.yamlName | toLower}}){{end}} |
```

Comment:
The last `if` expression seems to have two conditions.  What does that mean?

Reply:
If there is an elementKind then it's a map/slice so add an "s" because there's more then one of what ever type the map/slice is.  The second condition is to add a "(see below)" if the elementKind is equal to "object".  The end result is "map of objects (see below)"